Source/d_main.c

```diff
@@ -2303,6 +2246,11 @@ void D_DoomMain(void)
       debugfile = fopen(filename,"w");
     }
 
+  if (!demorecording)
+    I_AtExit(D_EndDoom, false);
+
+  TryRunTics();
```

Comment:
Why TryRunTics here?

Reply:
This is from Choco, but it does not apply to this PR, I staged it by accident.